docs/Guides-Routes.md

```diff
@@ -120,7 +124,11 @@ But now, we can also create routes for arbitrary user IDs. For example, if we wa
 ```
 window.addEventListener('popstate', () => {
   var userID = getQueryParamFromURI('userID', document.location.href);
-  var profileRoute = new ProfileRoute({userID});
-  // ...
+  var profileRoute = new ProfileRoute({userID: userID});
+  React.render(
+    <Relay.RootContainer
+      Component={UserProfile}
+      route={profileRoute}
+  />, container);
```

Comment:
Styling nit, can you indent `/>` and `);` to match the indentation of `<Relay.RootContainer` and `React.render(`, respectively? And line break all arguments (prefer all or none, I should add this to `CONTRIBUTING.md`).